### PR TITLE
283 Fix traces when creating/replacing/deleting workloads

### PR DESCRIPTION
**Pull request by krucod3**
Issues: #283 

# Definition of Done

The PR shall be merged only if all items mentioned in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [x] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

**Comment by maturar** [comment]
There is one unit test failing here. It is probably due to the change in this PR. @krucod3 could you check this, please?

File: agent/src/workload/workload_control_loop.rs [hunk 1]
```diff
@@ -347,8 +347,8 @@ impl WorkloadControlLoop {
             .await
         {
             Ok((new_workload_id, new_state_checker)) => {
-                log::debug!(
-                    "Created workload '{}' successfully.",
+                log::info!(
+                    "Successfully created workload '{}'.",
```
[the remark this answers is not in the document]
Comment (maturar) [reply agent/src/workload/workload_control_loop.rs:351]: I have checked the issue. It is in in the configuration. In the yaml. It is a bit confusing that the workload "hello2" exits after two seconds and it is configured to be always restarted. It leads to the situation that the Ankaios re-creates the workload again and again. I do not expect such behavior from the "Hello world" workload. I would suggest to change the example.

**Comment by krucod3** [comment]
@maturar, thanks for the fix!

**Review by maturar — APPROVED** [review]
LGTM :+1:

**Pull request merged**